examples/integrations/airflow/deepchecks_airflow_tutorial.py

```diff
@@ -0,0 +1,115 @@
+from datetime import datetime, timedelta
```

Comment:
1. Is it possible for the rst file to take "code blocks" as lines from here?
(so that in the document it will be a reference to line numbers instead of copy-paste of code...?)
Worth checking (and if so implementing) for finding a good and maintainable solution for the integration tutorial status (that is currently double...)
2. What's the purpose of this file?
(Is it quite easy to run it as-is? can we test it?
and do we think it's simpler than copy-pasting from the docs these code blocks?)

Reply:
For 1 - unfortunately, I can't see a good maintainable solution here. If we want explanations within the code it can be an ipynb, and then refer the doc page to a Colab link. If it's a simple Python script - then probably have it inside the RST. I know it's not maintainable, will have to decide what's the best way to do that.

2 - in order to get it tested we need airflow as a dependency, which I don't think will be a good idea. The script itself is quite easy to run assuming you have airflow installed.